libpod/runtime_ctr.go

```diff
@@ -44,6 +45,11 @@ type CtrCreateOption func(*Container) error
 // A true return will include the container, a false return will exclude it.
 type ContainerFilter func(*Container) bool
 
+// ExternalContainerFilter is a function to determine whether a container list is included
+// in command output. Container lists to be outputted are tested using the function.
+// A true return will include the container list, a false return will exclude it.
+type ExternalContainerFilter func(*contEntities.ListContainer) bool
```

Comment:
Why is this part of libpod? There is not a single field that depends on any libpod type so I don't think this should belong here.

Reply:
Yea, that's my failure I used `Runtime` receiver, I wanted to move filtering logic into separate function.

Let me keep it as separate non-exported function or do you want to put filtering logic into `GetContainers` function back?